tests/integration/nm/linux_bridge_test.py

```diff
@@ -22,24 +22,65 @@
 from libnmstate import nm
 from libnmstate.schema import LinuxBridge as LB
 
-from .testlib import mainloop
+from ..testlib import iproutelib
+from .testlib import mainloop_run
 
 
 BRIDGE0 = 'brtest0'
-ETH1 = 'eth1'
+PORT0 = 'eth1'
```

Comment:
how about also adding `port0_up`  = `eth1_up` to abstract the fixture for the port as well?